lib/nitcorn/http_request.nit

```diff
@@ -32,9 +35,6 @@ class HttpRequest
 	# Method of this request (GET or POST)
 	var method: String
 
-	# The host targetter by this request (usually the server)
-	var host: String
```

Comment:
This is old code, it was never set. The user code pick directly in the header map instead.